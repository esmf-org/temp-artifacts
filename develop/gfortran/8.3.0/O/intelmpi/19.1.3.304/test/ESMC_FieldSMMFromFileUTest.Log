2022-07-22 14:31:52
20220722 144746.629 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 144746.629 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220722 144746.629 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220722 144746.629 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220722 144746.629 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220722 144746.629 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 144746.629 INFO             PET0 Running with ESMF Version   : v8.4.0b07
20220722 144746.629 INFO             PET0 ESMF library build date/time: "Jul 22 2022" "14:30:29"
20220722 144746.629 INFO             PET0 ESMF library build location : /discover/nobackup/projects/sbu/mpotts/esmf-test/gfortran_8.3.0_intelmpi_O_develop/esmf
20220722 144746.629 INFO             PET0 ESMF_COMM                   : intelmpi
20220722 144746.629 INFO             PET0 ESMF_MOAB                   : enabled
20220722 144746.629 INFO             PET0 ESMF_LAPACK                 : enabled
20220722 144746.629 INFO             PET0 ESMF_NETCDF                 : disabled
20220722 144746.629 INFO             PET0 ESMF_PNETCDF                : disabled
20220722 144746.629 INFO             PET0 ESMF_PIO                    : disabled
20220722 144746.629 INFO             PET0 ESMF_YAMLCPP                : enabled
20220722 144746.629 INFO             PET0 Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41
20220722 144746.629 INFO             PET0 Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41NUMBER_OF_PROCESSORS 4
20220722 144746.636 ERROR            PET0 ESMF_IOScrip.F90:191 ESMF_OutputWeightFile Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET0 ESMF_IOScrip.F90:136 ESMF_SparseMatrixWrite Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET0 ESMF_Field_C.F90:1329 f_esmf_regridstorefile Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET0 ESMCI_Field.C:1601 ESMCI::Field::regridstorefile() Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET0 ESMC_Field.C:518 ESMC_FieldRegridStoreFile() Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 INFO             PET0 PASS ESMC_FieldRegridStoreFile test, ESMC_FieldSMMFromFileUTest.C, line 202
20220722 144746.636 INFO             PET0 PASS ESMC_FieldSMMStore from File test, ESMC_FieldSMMFromFileUTest.C, line 224
20220722 144746.636 INFO             PET0 PASS ESMC_FieldSMMStore From File validation, ESMC_FieldSMMFromFileUTest.C, line 244
20220722 144746.636 INFO             PET0 PASS ESMC_FieldRegridRelease, ESMC_FieldSMMFromFileUTest.C, line 274
20220722 144746.636 INFO             PET0 Ending Test, file ESMC_FieldSMMFromFileUTest.C, line 289
20220722 144746.636 INFO             PET0 Finalizing ESMF
20220722 144746.629 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 144746.629 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220722 144746.629 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220722 144746.629 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220722 144746.629 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220722 144746.629 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 144746.629 INFO             PET1 Running with ESMF Version   : v8.4.0b07
20220722 144746.629 INFO             PET1 ESMF library build date/time: "Jul 22 2022" "14:30:29"
20220722 144746.629 INFO             PET1 ESMF library build location : /discover/nobackup/projects/sbu/mpotts/esmf-test/gfortran_8.3.0_intelmpi_O_develop/esmf
20220722 144746.629 INFO             PET1 ESMF_COMM                   : intelmpi
20220722 144746.629 INFO             PET1 ESMF_MOAB                   : enabled
20220722 144746.629 INFO             PET1 ESMF_LAPACK                 : enabled
20220722 144746.629 INFO             PET1 ESMF_NETCDF                 : disabled
20220722 144746.629 INFO             PET1 ESMF_PNETCDF                : disabled
20220722 144746.629 INFO             PET1 ESMF_PIO                    : disabled
20220722 144746.629 INFO             PET1 ESMF_YAMLCPP                : enabled
20220722 144746.629 INFO             PET1 Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41
20220722 144746.629 INFO             PET1 Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41NUMBER_OF_PROCESSORS 4
20220722 144746.636 ERROR            PET1 ESMF_IOScrip.F90:191 ESMF_OutputWeightFile Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET1 ESMF_IOScrip.F90:136 ESMF_SparseMatrixWrite Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET1 ESMF_Field_C.F90:1329 f_esmf_regridstorefile Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET1 ESMCI_Field.C:1601 ESMCI::Field::regridstorefile() Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET1 ESMC_Field.C:518 ESMC_FieldRegridStoreFile() Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 INFO             PET1 PASS ESMC_FieldRegridStoreFile test, ESMC_FieldSMMFromFileUTest.C, line 202
20220722 144746.636 INFO             PET1 PASS ESMC_FieldSMMStore from File test, ESMC_FieldSMMFromFileUTest.C, line 224
20220722 144746.636 INFO             PET1 PASS ESMC_FieldSMMStore From File validation, ESMC_FieldSMMFromFileUTest.C, line 244
20220722 144746.636 INFO             PET1 PASS ESMC_FieldRegridRelease, ESMC_FieldSMMFromFileUTest.C, line 274
20220722 144746.636 INFO             PET1 Ending Test, file ESMC_FieldSMMFromFileUTest.C, line 289
20220722 144746.636 INFO             PET1 Finalizing ESMF
20220722 144746.629 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 144746.630 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220722 144746.630 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220722 144746.630 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220722 144746.630 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220722 144746.630 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 144746.630 INFO             PET2 Running with ESMF Version   : v8.4.0b07
20220722 144746.630 INFO             PET2 ESMF library build date/time: "Jul 22 2022" "14:30:29"
20220722 144746.630 INFO             PET2 ESMF library build location : /discover/nobackup/projects/sbu/mpotts/esmf-test/gfortran_8.3.0_intelmpi_O_develop/esmf
20220722 144746.630 INFO             PET2 ESMF_COMM                   : intelmpi
20220722 144746.630 INFO             PET2 ESMF_MOAB                   : enabled
20220722 144746.630 INFO             PET2 ESMF_LAPACK                 : enabled
20220722 144746.630 INFO             PET2 ESMF_NETCDF                 : disabled
20220722 144746.630 INFO             PET2 ESMF_PNETCDF                : disabled
20220722 144746.630 INFO             PET2 ESMF_PIO                    : disabled
20220722 144746.630 INFO             PET2 ESMF_YAMLCPP                : enabled
20220722 144746.630 INFO             PET2 Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41
20220722 144746.630 INFO             PET2 Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41NUMBER_OF_PROCESSORS 4
20220722 144746.636 ERROR            PET2 ESMF_IOScrip.F90:191 ESMF_OutputWeightFile Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET2 ESMF_IOScrip.F90:136 ESMF_SparseMatrixWrite Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET2 ESMF_Field_C.F90:1329 f_esmf_regridstorefile Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET2 ESMCI_Field.C:1601 ESMCI::Field::regridstorefile() Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET2 ESMC_Field.C:518 ESMC_FieldRegridStoreFile() Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 INFO             PET2 PASS ESMC_FieldRegridStoreFile test, ESMC_FieldSMMFromFileUTest.C, line 202
20220722 144746.636 INFO             PET2 PASS ESMC_FieldSMMStore from File test, ESMC_FieldSMMFromFileUTest.C, line 224
20220722 144746.636 INFO             PET2 PASS ESMC_FieldSMMStore From File validation, ESMC_FieldSMMFromFileUTest.C, line 244
20220722 144746.636 INFO             PET2 PASS ESMC_FieldRegridRelease, ESMC_FieldSMMFromFileUTest.C, line 274
20220722 144746.636 INFO             PET2 Ending Test, file ESMC_FieldSMMFromFileUTest.C, line 289
20220722 144746.636 INFO             PET2 Finalizing ESMF
20220722 144746.629 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 144746.629 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220722 144746.629 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220722 144746.629 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220722 144746.629 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220722 144746.629 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220722 144746.629 INFO             PET3 Running with ESMF Version   : v8.4.0b07
20220722 144746.629 INFO             PET3 ESMF library build date/time: "Jul 22 2022" "14:30:29"
20220722 144746.629 INFO             PET3 ESMF library build location : /discover/nobackup/projects/sbu/mpotts/esmf-test/gfortran_8.3.0_intelmpi_O_develop/esmf
20220722 144746.629 INFO             PET3 ESMF_COMM                   : intelmpi
20220722 144746.629 INFO             PET3 ESMF_MOAB                   : enabled
20220722 144746.629 INFO             PET3 ESMF_LAPACK                 : enabled
20220722 144746.629 INFO             PET3 ESMF_NETCDF                 : disabled
20220722 144746.629 INFO             PET3 ESMF_PNETCDF                : disabled
20220722 144746.629 INFO             PET3 ESMF_PIO                    : disabled
20220722 144746.629 INFO             PET3 ESMF_YAMLCPP                : enabled
20220722 144746.630 INFO             PET3 Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41
20220722 144746.630 INFO             PET3 Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41NUMBER_OF_PROCESSORS 4
20220722 144746.636 ERROR            PET3 ESMF_IOScrip.F90:191 ESMF_OutputWeightFile Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET3 ESMF_IOScrip.F90:136 ESMF_SparseMatrixWrite Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET3 ESMF_Field_C.F90:1329 f_esmf_regridstorefile Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET3 ESMCI_Field.C:1601 ESMCI::Field::regridstorefile() Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 ERROR            PET3 ESMC_Field.C:518 ESMC_FieldRegridStoreFile() Library needed by ESMF not present  - Internal subroutine call returned Error
20220722 144746.636 INFO             PET3 PASS ESMC_FieldRegridStoreFile test, ESMC_FieldSMMFromFileUTest.C, line 202
20220722 144746.636 INFO             PET3 PASS ESMC_FieldSMMStore from File test, ESMC_FieldSMMFromFileUTest.C, line 224
20220722 144746.636 INFO             PET3 PASS ESMC_FieldSMMStore From File validation, ESMC_FieldSMMFromFileUTest.C, line 244
20220722 144746.636 INFO             PET3 PASS ESMC_FieldRegridRelease, ESMC_FieldSMMFromFileUTest.C, line 274
20220722 144746.636 INFO             PET3 Ending Test, file ESMC_FieldSMMFromFileUTest.C, line 289
20220722 144746.636 INFO             PET3 Finalizing ESMF
