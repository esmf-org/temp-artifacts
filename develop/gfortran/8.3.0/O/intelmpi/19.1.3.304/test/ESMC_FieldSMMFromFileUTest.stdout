2022-07-22 14:31:52
srun: cluster configuration lacks support for cpu binding
Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41
Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41
Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41
Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41NUMBER_OF_PROCESSORS 4
Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41
Beginning Test, file ESMC_FieldSMMFromFileUTest.C, line 41NUMBER_OF_PROCESSORS 4
PASS ESMC_FieldRegridStoreFile test, ESMC_FieldSMMFromFileUTest.C, line 202
PASS ESMC_FieldRegridStoreFile test, ESMC_FieldSMMFromFileUTest.C, line 202
PASS ESMC_FieldSMMStore from File test, ESMC_FieldSMMFromFileUTest.C, line 224
PASS ESMC_FieldSMMStore From File validation, ESMC_FieldSMMFromFileUTest.C, line 244
PASS ESMC_FieldRegridStoreFile test, ESMC_FieldSMMFromFileUTest.C, line 202
PASS ESMC_FieldRegridStoreFile test, ESMC_FieldSMMFromFileUTest.C, line 202
PASS ESMC_FieldSMMStore from File test, ESMC_FieldSMMFromFileUTest.C, line 224
PASS ESMC_FieldSMMStore From File validation, ESMC_FieldSMMFromFileUTest.C, line 244
PASS ESMC_FieldRegridRelease, ESMC_FieldSMMFromFileUTest.C, line 274
PASS ESMC_FieldSMMStore from File test, ESMC_FieldSMMFromFileUTest.C, line 224
PASS ESMC_FieldSMMStore From File validation, ESMC_FieldSMMFromFileUTest.C, line 244
PASS ESMC_FieldRegridRelease, ESMC_FieldSMMFromFileUTest.C, line 274
Ending Test, file ESMC_FieldSMMFromFileUTest.C, line 289
PASS ESMC_FieldRegridRelease, ESMC_FieldSMMFromFileUTest.C, line 274
PASS ESMC_FieldSMMStore from File test, ESMC_FieldSMMFromFileUTest.C, line 224
Ending Test, file ESMC_FieldSMMFromFileUTest.C, line 289
PASS ESMC_FieldSMMStore From File validation, ESMC_FieldSMMFromFileUTest.C, line 244
PASS ESMC_FieldRegridRelease, ESMC_FieldSMMFromFileUTest.C, line 274
Ending Test, file ESMC_FieldSMMFromFileUTest.C, line 289
Ending Test, file ESMC_FieldSMMFromFileUTest.C, line 289
Ending Test, file ESMC_FieldSMMFromFileUTest.C, line 289 PET 1 Test Elapsed Time 58.316 msec.
Ending Test, file ESMC_FieldSMMFromFileUTest.C, line 289 PET 3 Test Elapsed Time 57.805 msec.
Ending Test, file ESMC_FieldSMMFromFileUTest.C, line 289 PET 0 Test Elapsed Time 58.335 msec.
Ending Test, file ESMC_FieldSMMFromFileUTest.C, line 289 PET 2 Test Elapsed Time 57.532 msec.
